docs/reference/FAQ.md

```diff
@@ -0,0 +1,63 @@
+---
+layout: default
+title: FAQ
+parent: Reference
+nav_order: 4
+permalink: /reference/faq/
+---
+
+- TOC
+{:toc}
+
+# What languages does ClusterFuzz support?
+
+ClusterFuzz definitely supports C, C++ when compiled with
+[clang](https://clang.llvm.org/). It should also support Rust and it probably
+supports other languages that can be compiled with an LLVM-based toolchain (e.g.
```

Comment:
I think that some statements on this page might be confusing. E.g. here we mention LLVM-based toolchain is a requirement (or a recommendation? "should also support" and "probably supports" sound weird?), but later on it becomes clear that a more important piece is the sanitizers. So, apparently you can do e.g. blackbox fuzzing with a target compiled with GCC and ASan, so LLVM toolchain is not a strong requirement.

I feel like we should think more on the structure of this page.

Reply:
> I think that some statements on this page might be confusing. E.g. here we mention LLVM-based toolchain is a requirement (or a recommendation? "should also support" and "probably supports" sound weird?),

The statements are intentionally a bit vague because we don't really know =)
I think CF supports rust because we have some rust fuzzers, but I'm not actually sure the support is fully there.

>but later on it becomes clear that a more important piece is the sanitizers. So, apparently you can do e.g. blackbox fuzzing with a target compiled with GCC and ASan, so LLVM toolchain is not a strong requirement.
> 
> I feel like we should think more on the structure of this page.

I've made the page less ambiguous. I initially wrote it so that it would scare people away from using GCC while letting brave souls know they could actually use it, I think the new version basically does the same while being less unclear.